src/components/PeopleFilters.tsx

```diff
@@ -25,68 +80,60 @@ export const PeopleFilters = () => {
       </div>
 
       <div className="panel-block">
-        <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
+        <div
+          className="
```

Comment:
It is recommended to use a single string for className instead of multiline string. You can write the className in one line.